paymentsheet-example/src/androidTest/java/com/stripe/android/lpm/TestKlarna.kt

```diff
@@ -18,21 +20,22 @@ internal class TestKlarna : BasePlaygroundTest() {
     ) { settings ->
         settings[CountrySettingsDefinition] = Country.US
         settings[CurrencySettingsDefinition] = Currency.USD
+        settings[AutomaticPaymentMethodsSettingsDefinition] = true
     }
 
-    @Ignore("Complex authorization handling required")
     @Test
-    fun testKlarna() {
-        testDriver.confirmNewOrGuestComplete(
+    fun testKlarnaInCustomFlow() {
+        testDriver.confirmCustom(
             testParameters = testParameters,
         )
     }
 
-    @Ignore("Complex authorization handling required")
     @Test
-    fun testKlarnaInCustomFlow() {
+    fun testKlarnaSetupIntentInCustomFlow() {
         testDriver.confirmCustom(
-            testParameters = testParameters,
+            testParameters = testParameters.copyPlaygroundSettings { settings ->
+                settings[CheckoutModeSettingsDefinition] = CheckoutMode.SETUP
+            },
```

Comment:
Is it possible to test PI+SFU?

Reply:
I think it is tested in the `testKlarnaSetupFutureUsage` method.